### PR TITLE
Add kunmingg as owner

### PR DESCRIPTION
**Pull request by kunmingg**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/839)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -9,6 +9,7 @@ approvers:
   - pdmack
   - ScorpioCPH
   - willb
+  - kunmingg
```
Comment (willingc) [inline OWNERS]: Great! Small nit - let's move this to keep alphabetical order. Thanks.

File: OWNERS [hunk 2]
```diff
@@ -6,11 +6,12 @@ approvers:
   - gaocegege
   - jlewi
   - lluunn
+  - kunmingg
```
Comment (ankushagarwal) [inline OWNERS]: Reorder alphabetically?

**Pull request merged**